zipa/module.py

```diff
@@ -0,0 +1,21 @@
+import sys
+
+from .magic import SelfWrapper
+
+
+def register_module(name):
+    self = sys.modules['zipa']
+    sys.modules[name] = SelfWrapper(self)
+
+
+class ModuleImporter(object):
+    def __init__(self, module_name):
+        self.name = module_name
+
+    def find_module(self, name, path=None):
+        if name.startswith('{}.'.format(self.name)):
```

Comment:
maybe it is worth trying __name__ here instead? dunno if it fits but it would easier to call the importer etc.